api/bags_api/src/main/scala/uk/ac/wellcome/platform/storage/bags/api/BagsApi.scala

```diff
@@ -123,14 +129,19 @@ trait BagsApi extends Logging {
             )
 
           case Right(manifests) =>
-            complete(
-              DisplayResultList(
-                context = contextURL.toString,
-                results = manifests.map {
-                  ResultListEntry(_)
-                }
+            val etagValue = manifests.map(_.idWithVersion).mkString("&")
```

Comment:
Makes me wonder – is the order stable here?

(Not necessarily needs a fix here, just wondering)

Reply:
Good point, should sort.